Directory.Build.props

```diff
@@ -29,7 +29,7 @@
     <PkgDir>$(RepoRoot)pkg/</PkgDir>
 
     <!-- Output directories -->
-    <BinDir Condition="'$(BinDir)'==''">$(RepoRoot)bin/</BinDir>
+    <BinDir Condition="'$(BinDir)'==''">$(RepoRoot)bin\</BinDir>
```

Comment:
Instead use NormalizeDirectory which normalizes the path for the host machine OS.

i.e:
```xml
$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'bin'))
```
that will add the correct trailing slash.

Reply:
Good call. I'm not sure I knew about that function.
Fixed.